config/nephe.yml

```diff
@@ -37,18 +37,13 @@ spec:
   - name: v1alpha1
     schema:
       openAPIV3Schema:
-        description: CloudEntitySelector is the Schema for the cloudentityselectors
-          API.
+        description: CloudEntitySelector is the Schema for the cloudentityselectors API.
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation
```

Comment:
Upgrade your nephe builder image.. You shoulnt be seeing this change

Reply:
Done